canonical-websites/partners.ubuntu.com
# Add new run script

Update the `./run` script to reflect the model now used by www.ubuntu.com. Also, add `test` script.

QA
---

Try out:

``` bash
./run  # Or "./run start"
./run --help
./run build
./run watch
./run test
./run clean
```

Particularly check database persistence:

1. Run the server with `./run`
2. Go to <http://127.0.0.1:8003/admin>, make a change, e.g. adding a partner
3. Kill the local server with `CTRL+c`.
4. Check no docker containers are still running - `docker ps` should be empty (unless you're running docker in other terminals).
5. `./run` and go to <http://127.0.0.1:8003/admin>, check your previous change is still there
7. Kill the local server with `CTRL+c`.
8. Delete the database with `./run clean`
9. `./run`, go to  <http://127.0.0.1:8003/admin>, check your previous change is gone
